Source/Addendum/AdViewUtils.swift

```diff
@@ -52,22 +49,22 @@ public final class AdViewUtils: NSObject {
         failure(error)
     }
     
-    static func findWebView(_ view: UIView, closure:(UIView) -> Bool) -> UIView? {
```

Comment:
Same... we are finding recursively for WebView so the naming is apt... lets keep it as is

Reply:
This function can be used for different purposes 

For example:
``` Swift
let view = self.findWebView(myView) { (subView) -> Bool in
    return subView is UIButton
}
```